packages/connect-web/conformance/known-failing-client.txt

```diff
@@ -0,0 +1,5 @@
+# These tests are failing as follows:
+# - in all browsers/environments with the callback client, cancellations are calling the end-of-stream callback with an undefined error if a call is cancelled.
+# - in firefox with the promise client, the error is actual error {code: 2 (unknown), message: "AbortError: The operation was aborted. "} does not match expected code 1 (canceled)
+**/server-stream/cancel-after-zero-responses
+**/server-stream/cancel-after-responses
```

Comment:
The failure with the callback client is expected, it's the same as in packages/connect-node/conformance/known-failing-client.txt

But why do we have a new failure with Firefox here? 🤔 The code did not change, so I don't understand why this uncovers a new issue with Firefox. Let's isolate this and file a bug.

Reply:
This isn't a new failure here either. I did the same as above -- moved it to a known-failing file rather than inlining it into the package.json command. 

For most of the conformance tests in connect-web, this known-failing file only applies to usage of the callback client. If you see in the package.json, it's only used in the second call for each browser. Firefox is the exception because we see the expected failures with the promise client but also the same tests fail with the callback client in the same fashion as the other browsers. So to summarize:

chrome: only fails with callback client bc of the undefined / swallowed error passed to the end-of-stream callback.
firefox: fails with both clients -- promise is the expected one re: the mismatched error code. callback is the same failure as chrome.
safari: same failure as chrome.